### PR TITLE
update DANIMESQ-xdai space

### PR DESCRIPTION
**Pull request by DaniellMesquita**
Fixed token address.

Signed-off-by: Daniell Mesquita <daniellmesquita@protonmail.com>

### CODE REVIEW

File: spaces/daniell-mesquita-xdai/index.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "token": "0x84d7fc44e4121adb74ef8d00a0f94cee7880d473",
+  "token": "0x95d7C55C260c18d0E7705236773C0b7942e1a0bF",
```
Comment (gokaiorg) [inline spaces/daniell-mesquita-xdai/index.json:2]: We are doing some changes for spaces settings, you can change your strategy token if you want but the root token value must stay like that.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — CHANGES_REQUESTED** [review]
We've made a major global update, please pull before commit again your changes.
You can use the documentation to help you.
https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Comment by DaniellMesquita** [comment]
So, root address is same from Layer1 token address?

**Pull request closed without merging**